lib/rouge/lexers/email.rb

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Email < RegexLexer
+      tag 'email'
+      aliases 'eml', 'e-mail'
+      filenames '*.eml'
+      mimetypes 'message/rfc822'
+
+      title "Email"
+      desc "An email message"
+
+      start do
+        push :fields
+      end
+
+      state :fields do
+        rule %r/[:]/, Operator, :field_body
+        rule %r/[^\n\r:]+/, Name::Tag
+        rule %r/[\n\r]/, Name::Tag
+      end
+
+      state :field_body do
+        rule(/(\r?\n){2}/) { token Text; pop!(2) }
```

Comment:
@smokris My reading of the RFC is that the fields are separated from the body by a single line that includes only a CRLF (see [section 2.1](https://tools.ietf.org/html/rfc2822#section-2.1)). Technically that should be represented as `\r\n` but since people writing at their keyboards are likely to just use `\n`, I made the `\r` optional.

Reply:
Yes, thanks.  (In my case, copy-pasting from macOS Mail and MailMate into GitLab, the linebreaks are simply `\n`, so supporting both CRLF and LF-only sounds good to me.)